cmd/skopeo/copy.go

```diff
@@ -72,6 +73,7 @@ See skopeo(1) section "IMAGE NAMES" for the expected format
 	flags.StringSliceVar(&opts.additionalTags, "additional-tag", []string{}, "additional tags (supports docker-archive)")
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Suppress output information when copying images")
 	flags.BoolVarP(&opts.all, "all", "a", false, "Copy all images if SOURCE-IMAGE is a list")
+	flags.Var(commonFlag.NewOptionalStringValue(&opts.multiArch), "multi-arch", `How to handle multi-architecture indexes (system, all, or index-only)`)
```

Comment:
```suggestion
	flags.Var(commonFlag.NewOptionalStringValue(&opts.multiArch), "multi-arch", `How to handle multi-architecture images (system, all, or index-only)`)
```

Reply:
Technically, its not a multi-architcture image, its an index of images (or manifest list, or fat manifest).

Do you think for skopeo users it would be better UX to gloss over that? I'm happy to go with it if you're sure :)

Reply:
Looking at your further comments, you seem sure, I'll go with it :)

Reply:
I was going to say “we don’t document `skopeo copy` to copy manifest/config/layers individually, it’s a tool to copy container images and the rest are implementation details” — except that the man page does happen to talk about the details :)

Overall, yes, I don’t think most users care, nor should they have to care. A multi-arch/single-arch image is a clear enough concept. Talking about OCI indices / schema2 manifest lists introduces terminology most users don’t care about, especially when those items can’t even stand alone (registries usually don’t allow uploading indices without the per-arch images).

Users of the index-only option do care; I don’t think we should optimize for that case, making the simpler cases harder to understand.